x-pack/plugins/security_solution/public/flyout/entity_details/host_right/index.tsx

```diff
@@ -107,6 +108,24 @@ export const HostPanel = ({
 
   const hasMisconfigurationFindings = passedFindings > 0 || failedFindings > 0;
 
+  const { data: vulnerabilitiesData } = useVulnerabilitiesPreview({
+    query: buildEntityFlyoutPreviewQuery('host.name', hostName),
+    sort: [],
+    enabled: true,
+    pageSize: 1,
+  });
+
+  const {
+    CRITICAL = 0,
+    HIGH = 0,
+    MEDIUM = 0,
+    LOW = 0,
+    UNKNOWN = 0,
+  } = vulnerabilitiesData?.count || {};
+
+  const totalVulnerabilities = CRITICAL + HIGH + MEDIUM + LOW + UNKNOWN;
+  const hasVulnerabilitiesFindings = totalVulnerabilities > 0;
```

Comment:
This looks a little bit too verbose. I wonder if we could make it shorter:

```suggestion
const hasVulnerabilitiesFindings = sum(Object.values(vulnerabilitiesData?.count || {}))  > 0;
```